android/app/src/main/kotlin/io/lantern/apps/AppsDataProvider.kt

```diff
@@ -41,6 +48,15 @@ class AppsDataProvider(
         return packageName == thisPackageName
     }
 
+    private fun isSystemApp(packageName: String): Boolean {
+        return try {
+            val appInfo = packageManager.getApplicationInfo(packageName, 0)
+            (appInfo.flags and ApplicationInfo.FLAG_SYSTEM) != 0
```

Comment:
One way to resolve the issue is to exclude system apps that can be updated since this indicates they're user-facing

```kotlin
(appInfo.flags and ApplicationInfo.FLAG_SYSTEM) != 0 && (appInfo.flags and ApplicationInfo.FLAG_UPDATED_SYSTEM_APP) != 0)
```

Reply:
Thanks for noticing this, I think its best to remove this check, Since all OS have different kinds of system apps, which will complicate things.